src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java

```diff
@@ -585,6 +596,31 @@ private List<Event> assembleEventTerms(JsonNode data, boolean dwc) {
     return List.of(event);
   }
 
+  private void setMinMaxMeterField(AbstractMeterTerm term, Location location, JsonNode data,
+      boolean dwc) {
+    var sanitizedValue = termMapper.retrieveTerm(term, data, dwc);
+    var doubleElevation = stringToDouble(term, sanitizedValue);
+    if (doubleElevation == null && sanitizedValue != null) {
+      if (term instanceof MaximumElevationInMeters) {
+        location.setDwcVerbatimElevation(
+            location.getDwcVerbatimElevation() + "| unparseable value maximumElevationInMeters: "
```

Comment:
I actually don't think it is possible that we run the regex and it hits and it wouldn't parse to a double. When the regex doesn't hit it will return the raw value. So I don't think we would lose any information here.